### PR TITLE
Exit cleanly if Travis has no secure environment variables

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/airtap/airtap/issues/66

Will squash.

### CODE REVIEW

**Comment by ralphtheninja** [comment]
This first commit should fail. Pushing up commit after to solve the failing test.

**Comment by ralphtheninja** [comment]
Hmm this was odd. It fails on node 9 for some reason but works on node 8.

File: bin/airtap.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env node
+
+// Prevent external PRs of airtap users to fail browser tests
+if (process.env.TRAVIS_SECURE_ENV_VARS === 'false') {
+  console.log('Skipping airtap due to no secure travis environment!')
```
Comment (yeskunall) [inline bin/airtap.js]: Nit: Travis. Only because this will be seen on the user-side. 😅

File: test/unit/airtap.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const exec = require('child_process').exec
+const airtap = path.resolve(__dirname, '../../bin/airtap.js')
+
+test('exits cleanly and does nothing if no secure travis env', t => {
+  const env = Object.assign({}, process.env, {
+    TRAVIS_SECURE_ENV_VARS: 'false'
+  })
+  exec(airtap, { env }, (err, stdout, stderr) => {
+    t.error(err, 'no error')
+    t.equal(stdout.trim(), 'Skipping airtap due to no secure travis environment!')
```
Comment (yeskunall) [inline test/unit/airtap.js]: I guess it's okay for now, but I don't like how the test depends on a hard-coded string. Do you agree as well? 😅

**Comment by ralphtheninja** [comment]
@yeskunall Added `messages.js`. Better? :smile:

**Comment by yeskunall** [comment]
@ralphtheninja yesss, it is! 🙌🏽 couldn't have done it any better! 😁

File: test/unit/airtap.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const path = require('path')
+const exec = require('child_process').exec
+const airtap = path.resolve(__dirname, '../../bin/airtap.js')
+const messages = require('../../lib/messages')
+
+test('exits cleanly and does nothing if no secure travis env', t => {
+  const env = Object.assign({}, process.env, {
+    TRAVIS_SECURE_ENV_VARS: 'false'
+  })
+  exec(airtap, { env }, (err, stdout, stderr) => {
```
Comment (vweevers) [inline test/unit/airtap.js]: To support Windows I think this should be `exec('node ${airtap}', ..)`. Opening an issue.

**Pull request merged**